app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewState.kt

```diff
@@ -1,10 +1,36 @@
 package com.wire.android.ui.home.conversations
 
+import android.os.Parcelable
 import androidx.compose.ui.text.input.TextFieldValue
 import com.wire.android.ui.home.conversations.model.Message
+import kotlinx.parcelize.Parcelize
 
 data class ConversationViewState(
     val conversationName: String = "",
     val messages: List<Message> = emptyList(),
-    val messageText : TextFieldValue = TextFieldValue("")
+    val messageText: TextFieldValue = TextFieldValue("")
 )
+
+/**
+ * Represents an attachment part of a message to be sent
+ */
+@Parcelize
```

Comment:
Why do we need to declare it as `@Parcelize` ?

Reply:
I don't think it's needed per se, but as in this class we are going to be handling probably big `ByteArray`, using the `Parcelable` implementation theoretically will have a better performance than plain serialization